CHANGELOG.md

```diff
@@ -2,6 +2,91 @@
 
 ## Release 2025-01-06
 
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/). This release is titled `v20250130`.
+
+### Announcements
```

Comment:
The open-source project Bridge to Kubernetes will be retired on April 30, 2025. For more information, please see the Bridge to Kubernetes [repository](https://github.com/Azure/Bridge-To-Kubernetes).